File: hw/nase/u3/hw21AnimalFarm/JavaScript/view/farmView.js
class FarmView {
    static showWelcome() {
        console.log("=".repeat(50));
        console.log("  FARM MANAGEMENT SYSTEM   ");
        console.log("=".repeat(50));
    }

    static showAnimalInfo(info) {
        console.log(`\n[Animal ID: ${info.id}] Tipo: ${info.type} | Raza: ${info.breed}`);
        console.log(` -> Edad: ${info.ageMonths} meses | Peso: ${info.weight} kg`);
        if (info.extra) {
            console.log(` -> Detalles: ${info.extra}`);
        }
    }

    static showActionResult(actionName, result) {
        console.log(` -> ${actionName}: ${result}`);
    }
}

module.exports = FarmView;

File: hw/nase/u3/hw21AnimalFarm/Python/view/farm_view.py
class FarmView:
    @staticmethod
    def show_welcome():
        print("=" * 50)
        print("  FARM MANAGEMENT SYSTEM  ")
        print("=" * 50)

    @staticmethod
    def show_animal_info(info: dict):
        print(f"\n[Animal ID: {info['id']}] Tipo: {info['type']} | Raza: {info['breed']}")
        print(f" -> Edad: {info['age_months']} meses | Peso: {info['weight']} kg")
        if info.get('extra'):
            print(f" -> Detalles: {info['extra']}")

    @staticmethod
    def show_action_result(action_name: str, result: str):
        print(f" -> {action_name}: {result}")